taxonomy-manager-rest/src/main/java/com/digirati/taxman/rest/Roles.java

```diff
@@ -0,0 +1,8 @@
+package com.digirati.taxman.rest;
+
+public final class Roles {
+
+    public static final String TAXMAN_ADMIN = "taxman-admin";
```

Comment:
Two points:
1. Typically I've seen parts of a role name separated by underscores rather than hyphens, I think we should do the same here; and
2. Any reason why this isn't an enum instead?

Reply:
Also, let's drop the `TAXMAN` part - we haven't branded the product yet so we should avoid all colloquial terminology we use in house.

Reply:
Happy to make the changes to the role name.

The reason I went with String constants over an enum is that the calling code needs a String, so if this was an enum the calling code (which is part of a Quarkus extension) wouldn't be restricted to the enum values. Given that the only thing this would need to do if it was an enum is provide a String, making it a class with String constants is a better way of doing it in this case.